nodejs/src/stream_manager.rs

```diff
@@ -0,0 +1,561 @@
+// Copyright Pravega Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::util::js_log;
+use neon::prelude::*;
+use pravega_client::client_factory::ClientFactory;
+use pravega_client_config::{ClientConfig, ClientConfigBuilder};
+use pravega_client_retry::retry_result::RetryError;
+use pravega_client_shared::*;
+use pravega_controller_client::ControllerError;
+use tracing::info;
+
+// An internal rust struct that holds the necessary info to perform actions on StreamManager.
```

Comment:
use /// to make it attach as a doc comment. (This makes it easier to dig into things in the doc browser.)